lib/src/logger_extension.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:mason_logger/mason_logger.dart';
+
+/// Extension on the Logger class for custom styled logging.
+extension LoggerX on Logger {
+  /// Log a message in the "created" style of the CLI.
+  void created(String message) {
```

Comment:
Minor but thoughts on calling this something like `pretty` or `emphasis`?

```suggestion
  void emphasis(String message) {
```

Reply:
I’m fine either way just was thinking about if we have another non created message with the same styling